src/tools/rest-frontend/README.md

```diff
@@ -420,11 +420,22 @@ Angular binding (i.e. use `<span translate="LOCA_KEY"></span>` in favor of
 #### Links ####
 
 For external links, the normal HTML `a`-tag has to be used (`<a href="..."></a>`).
+If the external link has the same base URL as the frontend (e.g. frontend is at
+`http://localhost/` and the link points to `http://localhost/rss/feed.rss`),
+the html tag `target` has to be added to the link with the desired value, e.g.
+`_self` to open the link in the same window/tab or `_blank` to use a new one.
+An example would be `<a href="http://localhost/rss/feed.rss" target="_self"></a>`.
+
 For internal links (that are links that lead to another sub-page of the website)
-the special router directive `ui-sref` should be used, as a normal link would
-trigger a page reload. The `ui-sref` directive works on state names and not on links,
+two options are available. It is possible to use the normal `href` HTML attribute
+or to use the special `ui-sref` attribute defined by the frontend router.
+The `ui-sref` directive works on state names and not on links,
 so if a sub-page like `<website-url>/docs/tutorials` exists, one cannot use
-`<a ui-sref="/docs/tutorials"></a>`. The state name for the tutorials page has to
+`<a ui-sref="/docs/tutorials"></a>`; the state name for the tutorials page has to
 be used, which is most likely `main.dyn.tutorials` if the tutorials section is based
 on the `structure.json.in`. The link (with a simple loca key) would look like
 `<a ui-sref="main.dyn.tutorials" translate="MENU.LINK.TUTORIALS"></a>` therefore.
+The `ui-sref` variant requires the HTML to be specially compiled though, what makes
+the usage of a normal `href` attribute easier in most scenarios.
+The following link does exactly the same as the last example with `ui-sref`:
+`<a href="/docs/tutorials" translate="MENU.LINK.TUTORIALS"></a>`.
```

Comment:
No, absolutely not - the ui.router and angular-translate are entirely different packages and do not interact with each other at all. The reasons I added it are: 
1) There is probably no (serious) angular app out there at all that does not use the translate module. Everyone who had angular in their hands for more than 5 minutes knows what I'm talking about.
2) Writing a HTML link into the documentation without link text would be strange and maybe more confusing. For sake of simplicity I change it to three points though, that leaves space for own thoughs.